operator/duties/scheduler.go

```diff
@@ -20,19 +22,34 @@ import (
 	"github.com/bloxapp/ssv/logging/fields"
 	"github.com/bloxapp/ssv/networkconfig"
 	"github.com/bloxapp/ssv/operator/duties/dutystore"
+	"github.com/bloxapp/ssv/operator/slotticker"
 	"github.com/bloxapp/ssv/protocol/v2/types"
 )
 
 //go:generate mockgen -package=mocks -destination=./mocks/scheduler.go -source=./scheduler.go
 
+var slotDelayHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "slot_ticker_delay_milliseconds",
```

Comment:
According to [Prometheus naming guidelines](https://prometheus.io/docs/practices/naming/):

> ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
> ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.

Also, other metrics in the project use the `_seconds` suffix, so we might need to consider making them consistent. On the other hand, I don't foresee any issues if we don't change it.